Cleanups before internal testing (#629)
0xPolygonMiden/miden-base: docs/introduction/welcome.md

Like private accounts, the Miden operator only tracks a commitment to notes in the public database. Users need to communicate note details to each other off-chain (via a side channel) in order to consume private notes in transactions.

#### Public accounts

> Polygon Miden will support public smart contracts like Ethereum. Code and state of those accounts will be visible to the network and anyone can execute transactions against them.

Reviewer: ```suggestion
Polygon Miden supports public smart contracts like Ethereum. The code and state of those accounts is visible to the network and anyone can execute transactions against them.
```

> #### Public notes

Reviewer: Avoid future tense.

> As with public accounts, public notes will also be supported. That means, the Miden operator will publicly store note data. Note consumption will not be private.

Reviewer: ```suggestion
As with public accounts, public notes are also supported. That means, the Miden operator publicly stores note data. Note consumption is not private.
```
